spk/pyload/src/dsm-control.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Package
+PACKAGE="pyload"
+DNAME="pyLoad"
+
+PIDFILE="/var/run/pyload.pid"
+PYLOADCORE="/usr/local/python/bin/python /usr/local/pyload/share/pyload/pyLoadCore.py --pidfile=${PIDFILE}"
+export PATH="/usr/local/pyload/bin:$PATH"
+
+case $1 in
+    start)
+        if [ -f /usr/syno/etc/ssl/ssl.crt/server.crt -a \
```

Comment:
Seeing as this is a one-time thing, it'd be best to move this to the installer code.
You could add an `enable SSL` step in the wizard if it's optional.